ui/app/styles/components/http-requests-bar-chart.scss

```
@@ -0,0 +1,20 @@
+.http-requests-bar-chart-container {
+  margin-top: $spacing-s;
+  margin-bottom: $spacing-m;
+  display: flex;
+}
+
+.http-requests-bar-chart {
+  margin: auto;
+  overflow: inherit;
+
+  .domain,
+  .tick > line {
+    stroke: $grey;
```

This one should be lighter
```suggestion
    stroke: $light-grey;
```